### ISSUE TITLE
Project idea: <lucky number generator> in <python>

### ISSUE THREAD

**Issue by hady67** [Atharv-Attri/HacktoberFest-Projects#16]
### What is a usage example of this? (We get somethings are pointless, yet still complex and fun to make, so we won't hold it against you if there is none.)

Checklist:
- [ ] This has not been made in the language I want to make it in
- [✅ ] I 100% believe this  meets quality standards

**Comment by hady67** [comment]
my project idea is to create a lucky number generator for the user when he/she enters their name.
it is for fun purposes and also if someone wants to know their lucky number !

**Comment by Atharv-Attri** [comment]
Please Explain how this would work and what strategies you will use.

**Comment by hady67** [comment]
it is a simple code as per me but its would be pretty fun to work on!
 i would use len for the number generation

**Comment by Atharv-Attri** [comment]
As in you will return the length of their name?

**Comment by hady67** [comment]
yes and add a random number to it with a rand generator

**Comment by Atharv-Attri** [comment]
I feel that might be too simple, see if there is some algorithm or something that generates lucky numbers, or find another way to make it more complex

**Comment by hady67** [comment]
can you suggest something ?

**Comment by hady67** [comment]
like i can add some more variables based on there dob, star sign , and take some values from them and add them with a random number.

**Comment by Atharv-Attri** [comment]
https://stackoverflow.com/questions/9018614/algorithm-to-find-lucky-numbers

You can try that.

**Comment by Atharv-Attri** [comment]
I don't think that using a random number is a good idea, since when people input their info, they should be getting the same number over and over again

**Comment by hady67** [comment]
okay yes you are right , a random generator wouldn't make sense if its a lucky number , so should i take some inputs from the user add specific values to them and the print their lucky number ?

**Comment by Atharv-Attri** [comment]
Honestly, sure. Just check some of the other merged PRs so you know what level of quality we expect

**Comment by hady67** [comment]
sure will do !
